proto/cacheclient.proto

```diff
@@ -448,7 +452,18 @@ message _ListEraseRequest {
   }
 }
 
-message _ListEraseResponse {}
+message _ListEraseResponse {
+  message _Found {
```

Comment:
@kvcache same story here w/rt backward compat, right?  Existing SDKs will just ignore this data and continue to operate as they did before, yes?